samples/query-features-with-arcade-expression/src/main/java/com/esri/arcgismaps/sample/queryfeatureswitharcadeexpression/components/QueryFeaturesWithArcadeExpressionViewModel.kt

```diff
@@ -92,15 +92,18 @@ class QueryFeaturesWithArcadeExpressionViewModel(application: Application) :
                     error.message.toString()
                 )
             }
+
+            // get the RPD Beats layer from the map's operational layers
+            policeBeatsLayer = arcGISMap.operationalLayers.firstOrNull { layer ->
+                layer.id == "RPD_Reorg_9254"
+            }
+
+            // update query state, map is ready for user interaction
+            _queryStateFlow.value = QueryState()
```

Comment:
Little Nitpick: I think it's a bit clearer to explicitly set this to `READY_TO_START` here rather than leave it as a default value

Reply:
I swithered a bit on passing `READY_TO_START` explicitly, but it does make things clearer. I'll add it back in.